Assets/MixedRealityToolkit/InputModule/Scripts/InputSources/SupportedInputInfo.cs

```diff
@@ -12,13 +12,23 @@ namespace MixedRealityToolkit.InputModule.InputSources
     public enum SupportedInputInfo
     {
         None = 0,
+        /// <summary>
+        /// Deprecated: use PointerPosition
+        /// </summary>
```

Comment:
is it possible to mark these obsolete?

Reply:
Aha! TIL that `[Obsolete]` exists. I'll pop an update up soon.